imagetest/test_suites/image_boot/setup.go

```diff
@@ -11,5 +11,17 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 	if err != nil {
 		return err
 	}
-	return vm.Reboot()
+	if err := vm.Reboot(); err != nil {
+		t.Skip("reboot workflow failed")
```

Comment:
oh this should not be a skip. skip is for when you want to disable a test suite on some condition. in this case, just return the err